packages/examples/src/colocated-arguments/colocated-arguments.ts

```diff
@@ -0,0 +1,56 @@
+import { Colocated, Choreography } from "@choreography-ts/core";
+
+type Locations = "alice" | "bob" | "carol" | "dave";
+
+const dualResponse: Choreography<
+  Locations,
+  [
+    Colocated<string, "alice" | "bob" | "dave">,
+    Colocated<string, "carol" | "bob" | "dave">
+  ], // Covariance works! Type Colocated<`"alice" | "bob" | "dave"> is a subtype of Colocated<"alice" | "bob"  | "dave" | "carol">!
+  | [
+      Colocated<string, "bob" | "alice" | "carol">,
+      Colocated<string, "dave" | "alice" | "carol">
+    ]
+  | []
+> = async ({ locally, peel, multicast }, [msgAlice, msgCarol]) => {
+  const aliceMsg = peel(msgAlice);
```

Comment:
wait a second, why is this valid? If this choreography is being executed on `Locations = "alice" | "bob" | "carol" | "dave"`, it should be illegal to peel `Colocated<string, "bob" | "alice" | "carol">` because it would be undefined at `dave`, no?